ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/custom-transit-switch-masquerade-subnet/cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-workflow.metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"path": "cucushift/installer/rehearse/vsphere/ipi/ovn/custom-transit-switch-masquerade-subnet/cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"jianlinliu",
+			"gpei",
+			"jinyunma",
+			"anuragthehatter",
+			"jechen0648",
+			"kuiwang02",
+			"liangxia"
+		],
+		"reviewers": [
+			"jianlinliu",
+			"gpei",
+			"jinyunma",
+			"anuragthehatter",
+			"jechen0648",
+			"kuiwang02",
+			"liangxia"
+		]
+	}
+}
```

Comment:
need to fix this last line

Reply:
They are auto generated metadata files when we do `make jobs`